README.md

```diff
@@ -19,238 +20,67 @@ This is the base Nerves System configuration for [QEMU](http://wiki.qemu.org/Mai
-
-
-### Hello, Network!
-
-Now you're ready to proceed to building and booting up the Nerves
-[hello_network](https://github.com/nerves-project/nerves-examples/tree/master/hello_network)
-example:
-
-    $ git clone https://github.com/nerves-project/nerves-examples.git
-
-    $ cd nerves-examples/hello_network
-
-    $ curl -OL https://github.com/nerves-project/nerves_system_qemu_arm/raw/develop/qemu-ifup.sh
-    $ curl -OL https://github.com/nerves-project/nerves_system_qemu_arm/raw/develop/qemu-ifdown.sh
-
-    $ NERVES_TARGET=qemu_arm mix deps.get
-
-    $ NERVES_TARGET=qemu_arm mix firmware
-
-    $ fwup -a -d _images/qemu_arm/hello_network.img -i _images/qemu_arm/hello_network.fw -t complete
-
-    $ sudo qemu-system-arm -M vexpress-a9 -smp 1 -m 256                    \
-        -kernel _build/qemu_arm/dev/nerves/system/images/zImage            \
-        -dtb _build/qemu_arm/dev/nerves/system/images/vexpress-v2p-ca9.dtb \
-        -drive file=_images/qemu_arm/hello_network.img,if=sd,format=raw    \
-        -append "console=ttyAMA0,115200 root=/dev/mmcblk0p2" -serial stdio \
-        -net nic,model=lan9118                                             \
-        -net tap,ifname=tap0,script=qemu-ifup.sh,downscript=qemu-ifdown.sh
-
-Once booted up into the IEX console, you can verify that network connectivity and
-DNS name resolution works by typing in:
-
-    iex(1)> HelloNetwork.test_dns
-    {:ok,
-     {:hostent, 'nerves-project.org', [], :inet, 4,
-      [{192, 30, 252, 154}, {192, 30, 252, 153}]}}
+# Build it
+mix deps.get
+mix firmware
 
-[![Screenshot](http://i.imgur.com/tjcVfHdh.jpg)](http://i.imgur.com/tjcVfHd.jpg)
+# Create the base image file. This is like a virtual SDCard for the emulator.
+# It just needs to be done once and then you can use over-the-air updates
+mix firmware.image
 
-## Installation
+# Generate a shell script to invoke qemu (only needs to be done once)
+mix nerves.gen.qemu_script
 
-If [available in Hex](https://hex.pm/docs/publish), the package can be installed as:
+# Start qemu
+./run-qemu.sh
 
-  1. Add nerves_system_qemu_arm to your list of dependencies in `mix.exs`:
+# It should open a window and boot to an iex prompt. Since it's the first
+# boot and the application data partition hasn't been formatted, you should
+# see some log messages about it being corrupt and then it being formatted.
 
-        def deps do
-          [{:nerves_system_qemu_arm, "~> 0.11.0"}]
-        end
+# At this point, feel free to make it edit to the Elixir code. For example,
+# have it IO.puts something.
 
-  2. Ensure nerves_system_qemu_arm is started before your application:
+# When you're ready, build, and run a firmware update.
+mix firmware
+mix firmware.push
 
-        def application do
-          [applications: [:nerves_system_qemu_arm]]
-        end
+# Alternatively, see the nerves_firmware_ssh project for how to update using
+# OpenSSH's ssh client so that things like password protected private keys
```

Comment:
> OpenSSH's ssh client so that things like password protected private keys
This reads a little funny. maybe 
see the nerves_firmware_ssh project for additional usage information such as using OpenSSH with password protected keys.